test/tools/yulopti.cpp

```diff
@@ -331,20 +485,25 @@ int main(int argc, char** argv)
 			return 1;
 		}
 
+		if (arguments.count("object"))
+			objectPath = arguments["object"].as<string>();
+
 		YulOpti yulOpti;
 		bool disambiguated = false;
+
 		if (!nonInteractive)
-			cout << input << endl;
+			yulOpti.parseAndPrint(input, objectPath);
```

Comment:
This helper is used only once and I think it's actually clearer to have it directly here since it's so simple:
```suggestion
			{
				yulOpti.parse(input, objectPath);
				yulOpti.printObject();
			}
```